elifepubmed/generate.py

```diff
@@ -364,6 +365,17 @@ def set_abstract(parent, poa_article, abstract_label_types):
         set_abstract_text(abstract_tag, '', '')
 
 
+def set_plain_language_summary(parent, poa_article):
+    "set an OtherAbstract tag to include the digest as plain-language-summary"
+    if hasattr(poa_article, 'digest') and poa_article.digest:
+        tag_converted_digest = poa_article.digest.replace(
```

Comment:
Any more standard way to strip tags? e.g. PHP has a specialized `strip_tags` function
Is `<p>` the only one that may appear, or the only one that needs to be stripped because e.g. `<b>` is supported?

Reply:
Good question! Answering in two parts.

Firstly, I found an example eLIfe XML that has `<italic>` tags in the digest, and those result in `<italic>` appearing here. So, a more complete cleaning of content should happen here... perhaps. It is something I'd like more feedback on from Melissa on whether this field can support any markup (so far, it does not gracefully support it, but an `<i>` tag might be better than no `<i>` tag when viewing it?)

Secondly, there could be some Python HTML or XML tag cleaning libraries or functions. I cannot remember for sure whether I looked. Typically in the places where I've used plain character replacements for tags, I either a) knew the tagging would be exactly as expected (e.g. '<p>' tag with no additional attributes), or b) it was a workable solution until an alternative edge case appeared, at which time it would be changed to work for the edge case. Where there are any plain string replacements are places where we've not observed an edge case in the content we support at this time.

Reply:
More detail about one of the existing methods in use is `remove_tag()` in `elifearitcle` library (https://github.com/elifesciences/elife-article/blob/develop/elifearticle/utils.py#L40). This is used already for abstracts to strip out tags we really do not want, but the inline markup tags like `<italic>` are left in. We could use that function again with a complete list of tags we want to strip out and it should be pretty effective in producing a text-only result from the fairly consistent abstract JATS XML.

Reply:
Don't want to introduce additional libraries that are not used already, good for the existing one. Make a list of intrusive tags including`<italic>` and strip that.

Reply:
As I discovered on further inspection, it is possible to support inline tagging. My first draft was too simplistic. The code now follows the method of adding XML to XML, as found in setting abstract and title content.

There's some room for refactoring, because the code to replace `<italic>` tags with `<i>` tags and other supported inline tagging (`<b>` and `<u>`) is repeated three times now in `generate.py`. I could possibly consolidate some of this into `utils.py` for some consistency. Your thoughts @giorgiosironi?